com.unity.ml-agents/Runtime/Actuators/IActuator.cs

```diff
@@ -3,7 +3,7 @@ namespace Unity.MLAgents.Actuators
     /// <summary>
     /// Abstraction that facilitates the execution of actions.
     /// </summary>
-    public interface IActuator : IActionReceiver
+    public interface IActuator : IActionReceiver, IHeuristicProvider
```

Comment:
Do we still need this to be a separate interface with only one method? Can it be part of IActionReceiver?

Reply:
Hmm, good point. I made IActionReceiver in order to help with backward compatibility.  Let me think about this.

Reply:
I think for now i'd like to leave it the way it is as it allows for indirection of actions.  There might be an actuator that you want to write that delegates actions to other action receivers.  maybe in a robotics use case or something.